### ISSUE TITLE
Visual Feedback Port for Anki 2.1

### ISSUE THREAD

**Issue by 2pascal** [glutanimate/review-feedback#1]
Hi,

What is the current status of the 2.1 port for Visual Feedback? I tried at use the code that's currently posted to see if any basic features are working, but it doesn't appear in my addon list. Do you have a current ETA? Thank you.

**Comment by glutanimate** [comment]
Visual Feedback should now work with Anki 2.1 (67c18e2eeef7c4a1c20833c759934fc33467a9db). I will be testing the add-on for the next 1-2 weeks, after which I'll publish the 2.1 release on AnkiWeb.

Thanks again for the report!

**Comment by snakecase** [comment]
Tested the add-on and I think it's ready to be updated on [ankiweb](https://ankiweb.net/shared/info/1749604199) for 2.1 support. I suggest setting **feedbackDuration** from 250 to 350 (feels much smoother) though and don't forget to change its name in Anki's Add-ons list to "Review Feedback" from "review_feedback".

Also a question: how did you make your zen icon's [transition](https://github.com/glutanimate/review-feedback/raw/master/screenshots/reviewer.gif) gradual? I tested 1.0.0 and there is no such transition for me, it just blinks in and out.

Thank you!
